Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -16,23 +24,48 @@ BatteryStateOfChargeService::~BatteryStateOfChargeService()
 
 double BatteryStateOfChargeService::totalAmpHoursUsed() const
 {
-    return 0.0;
+
+    return ampHour_;
 }
 
 bool BatteryStateOfChargeService::isCharging() const
 {
+
+    if(current_<0){
+
+        return true;
+    }
```

Comment:
fix up the formatting for if-statements, see the style guide if you are unsure

Reply:
example of correct if/else statement:
if (test_expression)
{
     stuff;
     more stuff;
}
else
{
     other stuff;
}